satpy/tests/reader_tests/test_seviri_l1b_icare.py

```diff
@@ -64,18 +65,15 @@ def get_test_content(self, filename, filename_info, filename_type):
         file_content['Brightness_Temperature/attr/add_offset'] = 0.
         file_content['Brightness_Temperature/shape'] = DEFAULT_FILE_SHAPE
 
-        # convert tp xarrays
+        # convert to xarrays
         from xarray import DataArray
         for key, val in file_content.items():
             if isinstance(val, np.ndarray):
                 attrs = {}
                 for a in ['_FillValue', 'scale_factor', 'add_offset']:
                     if key + '/attr/' + a in file_content:
                         attrs[a] = file_content[key + '/attr/' + a]
-                file_content[key] = DataArray(val, dims=('fakeDim0', 'fakeDim1'), attrs=attrs)
-        if 'y' not in file_content['Normalized_Radiance'].dims:
-            file_content['Normalized_Radiance'] = file_content['Normalized_Radiance'].rename({'fakeDim0': 'x',
-                                                                                              'fakeDim1': 'y'})
```

Comment:
Just to double check: Did you remove these lines intentionally?

Reply:
Yes, I copy-pasted the code from elsewhere and realise now that I don't actually need those lines here, can just explicitly set the dims to `x` and `y` on the line above.